### DISCUSSION TITLE
Powerbox P2-bus

### DISCUSSION THREAD

**Discussion by Gitarritzan** [iNavFlight/inav#8753] [Ideas]
Would it be possible to implement the Powerbox P2-bus protocol among the Receiver protocols?

**Comment by DzikuVx** [comment]
Could you provide more context and when and for whom this might be helpful?

**Reply by Gitarritzan** [reply]
Powerbox is a German manufacturer of a high-end radio control system. They are fairly new to the transmitter/receiver side of the business and they are a direct competitor to systems like Jeti and Futaba. They are probably aiming at the advanced side of RC in the first place and most people who use their system today are flying, jets, helicopters and gassers. I guess that you can say that they are in the opposite side of the spectrum to people who mostly fly FPV with more inexpensive Chinese electronics.

Powerbox has developed their own, two way, bus system called P2-bus and they claim it to be very fast and reliable. They also claim that their radio link is long range in the vicinity of 7-9 km. All this would of course make it very suitable for FPV.

To be honest, I can´t say that I believe that a lot of  FPV pilots of today would benefit from having the alternative to use the P2- bus protocol. But for those who would, if it was possible, it would probably be an excellent connection.
 
My own interest in the matter comes from that I just have migrated from the Jeti system to Powerbox and bought their newly released Atom transmitter to fly my jets.  Of course I immediately thought  of the possibility to fly my drones with the same system. I should also say that I don´t have anything to do with Powerbox besides being their customer.

But I realise that you can´t put in a lot of work to satisfy just me. On the other hand, you can´t blame a girl for trying.

Kind regards

Jannica
